docker-images/Dockerfile.python-connector

```diff
@@ -10,6 +11,11 @@ ARG BASE_IMAGE
 ARG CONNECTOR_NAME
 ARG EXTRA_PREREQS_SCRIPT=""
 
+# Install git and openssh-client for cloning repositories if needed
```

Comment:
Added (new) - this supports installing and testing connectors using dev versions of the CDK.